Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    react_on_rails (14.0.5)
+    react_on_rails (15.0.0.alpha.1)
```

Comment:
@Judahmeek did you self-review this change?

Reply:
I did.

I added this change here because I didn't want to make a PR just for the Gemfile.lock.

The Gemfile.lock should have actually have been updated either along with or right after the latest release.